cmd/api/src/api/v2/saved_queries_test.go

```diff
@@ -262,6 +263,321 @@ func TestResources_ListSavedQueries(t *testing.T) {
 	}
 }
 
+func TestResources_ListSavedQueries_OwnedQueries(t *testing.T) {
+	var (
+		mockCtrl  = gomock.NewController(t)
+		mockDB    = mocks.NewMockDatabase(mockCtrl)
+		resources = v2.Resources{DB: mockDB}
+	)
+	defer mockCtrl.Finish()
+
+	endpoint := "/api/v2/saved-queries"
+	userId, err := uuid2.NewV4()
+	require.Nil(t, err)
+
+	mockDB.EXPECT().ListSavedQueries(gomock.Any(), userId, gomock.Any(), gomock.Any(), 1, 10).Return(model.SavedQueries{
+		{
+			UserID:      userId.String(),
+			Name:        "myQuery",
+			Query:       "Match(n) return n;",
+			Description: "Public query description",
+		},
+	}, 1, nil)
+
+	if req, err := http.NewRequestWithContext(createContextWithOwnerId(userId), "GET", endpoint, nil); err != nil {
+		t.Fatal(err)
+	} else {
+		q := url.Values{}
+		q.Add("sort_by", "-name")
+		q.Add("name", "eq:myQuery")
+		q.Add("skip", "1")
+		q.Add("limit", "10")
+		q.Add("scope", "owned")
+
+		req.Header.Set(headers.ContentType.String(), mediatypes.ApplicationJson.String())
+		req.URL.RawQuery = q.Encode()
+
+		router := mux.NewRouter()
+		router.HandleFunc(endpoint, resources.ListSavedQueries).Methods("GET")
+
+		response := httptest.NewRecorder()
+		router.ServeHTTP(response, req)
+		require.Equal(t, http.StatusOK, response.Code)
```

Comment:
I always recommend using assert for your actual test criteria, as they will not fatally stop the test. In this case, the test has no logic after a single assert, but in the future there may be additional test cases or possible cleanup logic and changing these to asserts will help out the future engineer working on the test.